integrations/sklearn/Neptune-Scikit-learn.ipynb

```diff
@@ -0,0 +1,2048 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "QXGSKS1u0n8x",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "# Scikit-learn + Neptune"
```

Comment:
I think we are missing some parts where:

- we show that you can disable certain options 
- we show you can just use a specific function to log something. 

I'd add those to give a good view of this integration.

Reply:
done